openedx/core/lib/new_relic_utils.py

```diff
@@ -0,0 +1,24 @@
+"""
+Contains utilities for working with New Relic.
+
+These utilities will properly skip execution in an environment that does not use
+New Relic.
+
+"""
+
+try:
```

Comment:
@andy-armstrong: Due to this code, it seemed reasonable to make this a utility.  However, this means this would need to live in a core repo if we were to ever move Course Home to its own repo.

FYI: @jbarciauskas